native/acter/src/api/push/notification_item.rs

```diff
@@ -234,11 +234,34 @@ impl NotificationItemInner {
         a.reaction_key()
     }
 
-    pub fn new_date(&self) -> Option<UtcDateTime> {
+    pub fn utc_start(&self) -> Option<UtcDateTime> {
         let NotificationItemInner::Activity(a) = &self else {
             return None;
         };
-        a.new_date()
+        let content = a
+            .date_time_range_content()
+            .expect("utc start is available on date time range content");
+        content.start_new_val()
+    }
+
+    pub fn utc_end(&self) -> Option<UtcDateTime> {
+        let NotificationItemInner::Activity(a) = &self else {
+            return None;
+        };
+        let content = a
+            .date_time_range_content()
+            .expect("utc end is available on date time range content");
+        content.end_new_val()
+    }
+
+    pub fn due_date(&self) -> Option<String> {
+        let NotificationItemInner::Activity(a) = &self else {
+            return None;
+        };
+        let content = a
+            .date_content()
+            .expect("due date is available on date content");
```

Comment:
and once more

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2740/commits/5c725035d298f6fb64369163c56d3fcb3f6632b1